lib/tasks/data/reindex_explainers.rake

```diff
@@ -0,0 +1,60 @@
+namespace :check do
+    namespace :data do
+      # bundle exec rake check:data:reindex_explainers
+      desc 'Reindex all explainers in the system'
+      task :reindex_explainers => [:environment] do
+        old_logger = ActiveRecord::Base.logger
+        ActiveRecord::Base.logger = nil
+  
+        errors = []
+        current_time = Time.now
+        event_id = ENV['EVENT_ID'] || Digest::MD5.hexdigest("reindex_all_explainers_#{current_time.to_i}")
```

Comment:
Looks like you copy-and-pasted this line from some other rake task but it's not used for anything here. So please delete this line.